app/helpers/application_helper.rb

```diff
@@ -34,6 +34,13 @@ def date_string(start_date, end_date)
     result
   end
 
+  # Returns the time with the conference's timezone
+  def set_offset(time, timezone)
```

Comment:
I think it is not compulsory to use it, So should I use the first approach or the second ?